### PR TITLE
Friday Announcements 2021 06 25

### PR DESCRIPTION
**Pull request by cokiengchiara**
### Please remember to

- [x] Give the PR a meaningful title -- it will be shown in the release notes

---

@marians , please add for UIs and Docs :)

### CODE REVIEW

File: src/content/changes/highlights/2021-06-25.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+- https://github.com/orgs/giantswarm/teams/sig-product
+---
+
+## General
+
+As we aim to align our terminology with the Kubernetes project where possible, we started replacing the term _master node_ with _control plane node_ in our communication and user interfaces.
+
+## Managed apps
+
+[EFK v0.5.3](https://docs.giantswarm.io/changes/managed-apps/efk-stack-app/v0.5.3/) adds support to override the Kibana image registry, increasing flexibility. For example, allowing customers to install their own plugins, set default configs, etc.
```
Comment (cokiengchiara) [inline src/content/changes/highlights/2021-06-25.md:16]: @QuentinBisson , @giantswarm/app-squad-efk - please check if OK w you :)

Comment (QuentinBisson) [reply src/content/changes/highlights/2021-06-25.md:16]: Good for me

**Review by marians — APPROVED** [review]
I'm done. Thanks!

File: src/content/changes/highlights/2021-06-25.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+- https://github.com/orgs/giantswarm/teams/sig-product
+---
+
+## General
+
+As we aim to align our terminology with the Kubernetes project where possible, we started replacing the term _master node_ with _control plane node_ in our communication and user interfaces.
+
+## Managed apps
+
+[EFK v0.5.4](https://docs.giantswarm.io/changes/managed-apps/efk-stack-app/v0.5.4/) adds a option to clean stale lock files in an init container. In some cases lock files are not cleaned up automatically when NFS is used for storage. In those cases Elasticsearch can't start.
```
Comment (cokiengchiara) [inline src/content/changes/highlights/2021-06-25.md:16]:
```suggestion
[EFK v0.5.4](https://docs.giantswarm.io/changes/managed-apps/efk-stack-app/v0.5.4/) adds an option to clean stale lock files in an init container. This addresses some cases when lock files are not cleaned up automatically (when NFS is used for storage) and Elasticsearch cannot start.
```

**Comment by marians** [comment]
Please ignore the validate / front-matter check. I'll have to mute the `Each page should have a description` check for the entire `changes` section.

**Pull request merged**